design-documents/graph-ql/coverage/search.md

```diff
@@ -0,0 +1,79 @@
+# Queries
+
+```graphql
+type Query {
+    multiSearch(phrase: String!, productSize: Int = 10): MultiSearchResponse!
+    #Filter supports multiple clauses which will be wrapped in logical AND operator
+    productSearch(phrase: String!, filter: [SearchClauseInput], sort: ProductSearchSortInput): ProductSearchResponse!
+}
+
+input ProductSearchSortInput
+{
+    relevance: SortEnum
```

Comment:
How will sort by relevance and position in same time will work?

Reply:
https://www.elastic.co/guide/en/elasticsearch/reference/6.8/search-request-sort.html#search-request-sort - most of engines support sorting by multiple fields. Some combinations may not have any business value, but there is no technical issues